rviz_rendering/CMakeLists.txt

```diff
@@ -175,4 +175,6 @@ list(APPEND ${PROJECT_NAME}_CONFIG_EXTRAS
   "${CMAKE_CURRENT_SOURCE_DIR}/src/cmake/register_rviz_ogre_media_exports_hook-extras.cmake"
   "${CMAKE_CURRENT_SOURCE_DIR}/src/cmake/register_rviz_ogre_media_exports.cmake")
 
+set_target_properties(rendering_example PROPERTIES INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
@wjwwood We need this for the linker to succeed. I remember that you do not need this. We are not quite sure what is causing this different behavior.

Reply:
I've actually got `rendering_example` disabled in the use pluginlib branch, maybe we should just get rid of the `rendering_example`.